metabase/metabase
# Remove unneeded driver deprecation messages

Epic: https://github.com/metabase/metabase/issues/17520

1. Makes the "deprecation switch" behavior not show up on the Setup page
2. Hides the "Get Help" card when deprecated driver is chosen

How to test setup changes:
1.  Run fresh Metabase instance
2. During setup, select a driver that supersedes another one, e.g. `BigQuery`
3. No message about driver deprecation should be shown

How to test admin changes:
1. Run Metabase
2. Navigate to Admin -> Databases -> Create
3. Select a driver that supersedes another one, e.g. `BigQuery`
4. Check that the help block with a link to the docs is shown
5. Check that the driver deprecation message is visible
6. Click "find it here" link, it should select the deprecated driver
7. Check that the help block is now hidden
